recipes/scons/all/conanfile.py

```diff
@@ -19,12 +21,23 @@ class SConsConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def configure(self):
+        # Detect a frozen conan PyInstaller
+        if getattr(sys, "frozen", False):
+            raise ConanException("This recipe requires a python interpreter and does not support a frozen conan installation. This can be fixed with a cpython recipe.")
```

Comment:
```suggestion
            raise ConanInvalidConfiguration("This recipe requires a python interpreter and does not support a frozen conan installation. This can be fixed with a cpython recipe.")
```

ConanException breaks the build